pkg/chart/chart.go

```diff
@@ -239,11 +248,19 @@ func fetchChart(netClient *HTTPClient, chartURL string, load LoadChart) (*chart.
 	if err != nil {
 		return nil, err
 	}
-	return load(bytes.NewReader(data))
+	chartV2, err := helm2loader.LoadArchive(bytes.NewReader(data))
+	if err != nil {
+		return nil, err
+	}
+	chartV3, err := helm3loader.LoadArchive(bytes.NewReader(data))
```

Comment:
I'd probably do something like `lol := bytes.NewReader(data)` and then use `lol` twice (assuming it typechecks).

Reply:
It does look like it should be DRYer, but `bytes.NewReader()` returns a `Reader` interface, which doesn't have a seek. So it can only be read once. I'd have to convert it to a `SectionReader` (so I can call `Seek(0)` after the first `LoadArchive`), but it ends up more complicated than this. Try it out (and let me know if you find a better option :)

Reply:
Hehe, I suspected there could be some black magic like that going on. I'm absolutely fine with the current implementation. :smile: